src/components/Button/styles.scss

```diff
@@ -217,3 +222,39 @@
 .rainbow-button--shaded {
     box-shadow: $shadow-1;
 }
+
+.rainbow-button--content-hidden {
```

Comment:
I think that .rainbow-button_content--hidden follow the class convention since content is a subcomponent not a modifier.